### PR TITLE
Allow filtering by name in status

### PR DESCRIPTION
**Pull request by 12pm**
Easier to find players when the server is full

### CODE REVIEW

File: src/engine/client/client.cpp [hunk 1]
```diff
@@ -3749,7 +3749,7 @@ void CClient::RegisterCommands()
 	m_pConsole->Register("ban", "s[ip|id] ?i[minutes] r[reason]", CFGFLAG_SERVER, 0, 0, "Ban player with ip/id for x minutes for any reason");
 	m_pConsole->Register("unban", "s[ip]", CFGFLAG_SERVER, 0, 0, "Unban ip");
 	m_pConsole->Register("bans", "", CFGFLAG_SERVER, 0, 0, "Show banlist");
-	m_pConsole->Register("status", "", CFGFLAG_SERVER, 0, 0, "List players");
+	m_pConsole->Register("status", "?s[name]", CFGFLAG_SERVER, 0, 0, "List players containing name or all players");
```
Comment (fokkonaut) [inline src/engine/client/client.cpp]: r

Comment (fokkonaut) [reply src/engine/client/client.cpp]: also, write "List player", not player*s*

Comment (def-) [reply src/engine/client/client.cpp]: agreed, then you don't need to quote names. rest looks fine.

Comment (12pm) [reply src/engine/client/client.cpp]: done

**Pull request merged**